subprojects/packagefiles/netstring-c/meson.build

```diff
@@ -0,0 +1,35 @@
+project(
+  'netstring-c',
+  'c',
+  version: '0.0.0',
+  license : 'Unlicense',
+  meson_version: '>= 0.55',
```

Comment:
This was wrong, it only needed 0.40 -- you can test this by setting the version to `>=0.00` and checking which versions it complains about.

Reply:
It was a somewhat random choice to be honest, didn't now about the trick with `>=0.00`, will use it next time, thanks!

Reply:
No problem -- fixed and pushed.